packages/react-dev-utils/FileSizeReporter.js

```diff
@@ -98,6 +98,7 @@ function printFileSizesAfterBuild(
 function removeFileNameHash(buildFolder, fileName) {
   return fileName
     .replace(buildFolder, '')
+    .replace(/\\/g, '/')
```

Comment:
I think it would be better to just support both path separators in the below regex, what do you think?

Reply:
Sorry, I should've been clearer.
Is this because `asset.name` always returns paths with forward slashes (`/`)? If so, this patch makes perfect sense!